traveler-ios-ui/TravelerKitUI/ViewControllers/Parking/ParkingQueryViewController.swift

```diff
@@ -64,9 +64,15 @@ open class ParkingQueryViewController: UIViewController {
         case 0:
             mapContainerView.isHidden = false
             listContainerView.isHidden = true
+            segmentContainerView.backgroundColor = .clear
         case 1:
             mapContainerView.isHidden = true
             listContainerView.isHidden = false
+            if #available(iOS 13.0, *) {
+                segmentContainerView.backgroundColor = .secondarySystemBackground
+            } else {
+                segmentContainerView.backgroundColor = UIColor(red: 242.0/255.0, green: 242.0/255.0, blue: 247.0/255.0, alpha: 1.0)
```

Comment:
Shouldn't be setting any custom colors

Reply:
The corresponding color in iOS 12.0 and lower of secondarySystemBackground is that custom color. Should I use lightGray instead?

Reply:
Why do you need to toggle between the background colours in the first place?